workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/LaunchArguments.cs

```diff
@@ -27,7 +27,7 @@ public static class LaunchArguments
             }
             catch (Exception e)
             {
-                UnityEngine.Debug.LogException(e);
+                UnityEngine.Debug.LogWarning($"Failed to retrieve launch arguments: {e}");
```

Comment:
Exception -> Warning? What's the context here?

Reply:
this will always fail, if you run this in the editor after switching to Android as current platform

therefore, it is expected behaviour to fail depending on what exactly you do and it shouldn't throw an exception imo

Reply:
Should this function even be able to be called in the editor then if it will always fail?

Reply:
fair point. we could also change the part where it's called to only get called if android and not editor. or wrap this part here around an `#if UNITY_EDITOR`, which I would slightly prefer as it makes it safe to call for users that are not familiar with the code.